modules/storages/app/controllers/storages/admin/project_storages_controller.rb

```diff
@@ -169,6 +171,6 @@ def available_storages
     Storages::Storage
       .visible
       .not_enabled_for_project(@project)
-      .configured
+      .select(&:configured?)
```

Comment:
🟡 Switching to a ruby instance method is less performant and could become an issue if the number of storages are many (this is unlikely), as they are loaded into memory. Most importantly we also loose out on (lazy) chain able scopes- which is handy for forming queries.

I see I missed a discussion in https://github.com/opf/openproject/pull/13619#discussion_r1315762693 with @mereghost 😬

Reply:
I do agree with you @akabiru, but as you noted we the performance hit shouldn't be that terrible here.

If push comes to shove we could store the configured status in the DB to allow for easy queries, rather than trying to make crazy unions or cross joins.

Reply:
That is the point, we can reintroduce it, but it probably would require a db migration. This solution right now is the "simple" one. As the PR was not about the scope in the very beginning, but it stumbled across it to make one drive storages work at all, any improvements to the scope should come to a follow up PR.

If you already have a great idea, @akabiru, I'd be very intrigued to see it :D